automation/tests-container-utils.sh

```
@@ -13,10 +13,20 @@ function container_exec {
 }
 
 function container_exec_with_retry {
-    time ${CONTAINER_CMD} exec $USE_TTY -i $CONTAINER_ID \
-        /bin/bash -c "cd $CONTAINER_WORKSPACE && $1" ||
-    time ${CONTAINER_CMD} exec $USE_TTY -i $CONTAINER_ID \
-        /bin/bash -c "cd $CONTAINER_WORKSPACE && $1"
+    local retries=5
+    local i
+    for ((i=1; i<=retries; i++)); do
+        if time ${CONTAINER_CMD} exec $USE_TTY -i $CONTAINER_ID \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)